src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjects/DependentProjectsFinder.cs

```diff
@@ -0,0 +1,320 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this was a move, but appears to git to be so rerwritten as to be effectively all new.